docs/resources/Channel.md

```diff
@@ -277,7 +277,7 @@ Represents a message sent in a channel within Discord.
 | tts                         | boolean                                                                                                                                   | whether this was a TTS message                                                                                                                                                                                                                                          |
 | mention_everyone            | boolean                                                                                                                                   | whether this message mentions everyone                                                                                                                                                                                                                                  |
 | mentions                    | array of [user](#DOCS_RESOURCES_USER/user-object) objects                                                                                 | users specifically mentioned in the message                                                                                                                                                                                                                             |
-| mention_roles               | array of [role](#DOCS_TOPICS_PERMISSIONS/role-object) object ids                                                                          | roles specifically mentioned in this message                                                                                                                                                                                                                            |
+| mention_roles               | array of snowflakes                                                                                                                       | roles specifically mentioned in this message                                                                                                                                                                                                                            |
```

Comment:
```suggestion
| mention_roles               | array of snowflakes                                                                                                                       | [roles](#DOCS_TOPICS_PERMISSIONS/role-object) specifically mentioned in this message                                                                                                                                                                                                                            |
```
I think the reference should be kept, maybe in a way like this (same below)...

Reply:
I added "[role](#DOCS_TOPICS_PERMISSIONS/role-object) object ids"